source/includes/steps-favorite-connection-edit.yaml

```diff
@@ -7,12 +7,14 @@ content: |
      :figwidth: 600px
      :alt: Saved Favorite Connection
   
-  To :guilabel:`Copy` or :guilabel:`Remove` a Favorite Connection, 
-  hover over the target connection and click the :icon-fa5:`ellipsis-h` 
-  button.
+  To :guilabel:`Copy Connection String`, :guilabel:`Duplicate`, or 
+  :guilabel:`Remove` a Favorite Connection, hover over the target connection 
+  and click the :icon-fa5:`ellipsis-h` button.
   
-  - Copying a connection creates another Favorite Connection with the 
-    same configuration. 
+  - Copying a connection string saves the connection string to your clipboard. 
```

Comment:
I wonder if copying a connection string should be part of the `Edit a Favorite Connection` section.

Reply:
I did add the ability to copy a connection string to the main Connect to MongoDB page as part of this [PR](https://github.com/mongodb/docs-compass/pull/399) so that may suffice in terms of showing users that feature?